src/main/fc/cli.c

```diff
@@ -1945,7 +1945,8 @@ static void cliFeature(char *cmdline)
         for (uint32_t i = 0; ; i++) {
             if (featureNames[i] == NULL)
                 break;
-            cliPrintf(" %s", featureNames[i]);
+            if (strcmp(featureNames[i], "UNUSED") != 0)
```

Comment:
This is error prone. Use a define to make sure the right keyword is used.
Even better, use `""` (an empty string) to compare against, to optimise code size.